### PR TITLE
on error name back-compat with odbc

### PR DESCRIPTION
**Pull request by JyotikaGargg**
ONERROR -> replaced with on error for naming connvention matching with ODBC

### CODE REVIEW

File: pkg/sqlcmd/commands.go [hunk 1]
```diff
@@ -103,7 +103,7 @@ func newCommands() Commands {
 			isSystem: true,
 		},
 		"ONERROR": {
-			regex:  regexp.MustCompile(`(?im)^[\t ]*?:?ONERROR(?:[ \t]+(.*$)|$)`),
+			regex:  regexp.MustCompile(`(?im)^[\t ]*?:?ON ERROR(?:[ \t]+(.*$)|$)`),
```
Comment (apoorvdeshmukh) [inline pkg/sqlcmd/commands.go:106]: Does it also handle lowercase?
Should we have one of the unit tests in lowercase variant of this command?

Comment (shueybubbles) [reply pkg/sqlcmd/commands.go:106]: The (?im) modifier make the expression case insensitive

See regex101.com for a great interactive tool to play with regex

**Review by shueybubbles — APPROVED** [review]
:shipit:

**Review by stuartpa — APPROVED** [review]
:shipit:

**Pull request merged**